docs/computed-properties.md

```diff
@@ -1,6 +1,6 @@
 Computed properties are a way to create "derived" properties in Livewire. Like accessors on an Eloquent model, computed properties allow you to access values and cache them for future access during the request.
 
-Computed properties are particularly useful in combination with a component's public properties.
```

Comment:
I would revert this as it reads better to me with the "a".

Reply:
Seems a little bit off to me to write about plurals with an 'a' in between

Reply:
"componet's" is possessive, if it was "components" then yeah I'd agree 🙂